global/thanos-global/values.yaml

```
@@ -71,3 +71,12 @@ thanos:
   
   alerts:
     prometheus: kubernetes
+
+  ruler:
+    enabled: true
+    loglevel: warn
+    evaluationInterval: 30s
```

I'm not sure if we want to do this that often? I would start with `60s`. 
```suggestion
    evaluationInterval: 60s
```